### ISSUE TITLE
Compatible with tslint rules

### ISSUE THREAD

**Issue by nelson6e65** [keithamus/sort-package-json#70]
```tslint.json```

The `rules` are not sorted alphabetically, only the first level keys. 😢

**Comment by keithamus** [comment]
Thanks for the issue @nelson6e65, PRs welcome!

**Comment by nelson6e65** [comment]
I'll try. Any advise?

**Comment by keithamus** [comment]
The `sortSubKey` function within this codebase is used to sort the items of a key within the JSON, so adding `sortSubKey('rules')` would work I believe.

**Comment by fisker** [comment]
We should focus on sorting `package.json`.

**Comment by keithamus** [comment]
Yes perhaps we can extract a lot of the core functionality here and make a new `sort-tsconfig` package.

**Comment by nelson6e65** [comment]
> Yes perhaps we can extract a lot of the core functionality here and make a new `sort-tsconfig` package.

That would be great!

But... have different tools.... Hmm... Maybe split it in internal modules and detect the "format" automatically from filename or and explicit `--type`.

**Comment by nelson6e65** [comment]
Well, tslint becomes deprecated, but still would be nice to sort another `.json` config files with the same base cli.
